README.md

```diff
@@ -1,19 +1,122 @@
+
+		<Frame x:Name="RootNavigationFrame" />
+
+		<splash:ExtendedSplashScreen x:Name="AppExtendedSplashScreen" />
+	</Grid>
+</UserControl>
+```
+
+In the code behind of the user control you can set the splashscreen for UWP, and make a reference to the shell in Instance and make a public property to represent the ExtendedSplashscreen.            
+```C#
+public sealed partial class Shell : UserControl
+	{
+		public static Shell Instance { get; private set; }
+
+		public Shell(LaunchActivatedEventArgs e)
+		{
+			this.InitializeComponent();
+
+			Instance = this;
+
+#if WINDOWS_UWP
+			AppExtendedSplashScreen.SplashScreen = e?.SplashScreen;
+#elif WINDOWS
+			AppExtendedSplashScreen.SplashScreen = e?.UWPLaunchActivatedEventArgs.SplashScreen;
+#endif`
+
+			NavigationFrame.Navigate(typeof(MainPage), e.Arguments);
+		}
+
+		public IExtendedSplashScreen ExtendedSplashScreen => this.AppExtendedSplashScreen;
+
+		public Frame NavigationFrame => this.RootNavigationFrame;
+	}
+```
+
+You can reference the instance property later for navigation and you can dismiss with this line of code: 
+
+```c#
+  		Shell.Instance.ExtendedSplashScreen.Dismiss();
+```
+
+Then, all you need to do is to set the style 
+```xml
+<Style x:Key="DefaultExtendedSplashScreenStyle"
+		   TargetType="splash:ExtendedSplashScreen">
+		<Setter Property="Template">
+			<Setter.Value>
+				<ControlTemplate TargetType="splash:ExtendedSplashScreen">
+					<Grid x:Name="RootGrid">
+						<VisualStateManager.VisualStateGroups>
+							<VisualStateGroup x:Name="SplashScreenStates">
+								<VisualState x:Name="Normal" />
+								<VisualState x:Name="Dismissed">
+									<Storyboard>
+										<ObjectAnimationUsingKeyFrames Storyboard.TargetName="RootGrid"
+																	   Storyboard.TargetProperty="Visibility">
+											<DiscreteObjectKeyFrame KeyTime="0:0:0.150"
+																	Value="Collapsed" />
+										</ObjectAnimationUsingKeyFrames>
+										<DoubleAnimation Storyboard.TargetName="RootGrid"
+														 Storyboard.TargetProperty="Opacity"
+														 To="0"
+														 Duration="0:0:0.150" />
+									</Storyboard>
+								</VisualState>
+							</VisualStateGroup>
+						</VisualStateManager.VisualStateGroups>
+
+                        <!-- Here you can put the content you want in your ExtendedSplashscreen. The Content presenter here is the splashscreen that you have. You can put anything you want below it to make it appear over the splashscreen. -->
+						<ContentPresenter x:Name="SplashScreenPresenter" />
+
+						<Rectangle Fill="Red"
+								   Height="100"
+								   Width="100" />
+					</Grid>
+				</ControlTemplate>
+			</Setter.Value>
+		</Setter>
+	</Style>
+```
 
 ## Features
+Show() makes the splashscreen become visible.
+
+Dismiss() dismisses the splashscreen.
+
+`<ContentPresenter x:Name="SplashScreenPresenter" />`
+
+ In the style of the ExtendedSplashScreen shows the splashscreen for android, IOS and UWP.
```

Comment:
I don't understand this comment. Isn't it already mention above in the xaml example?

Reply:
Yeah, you're right